src/components/time-entry/TimeEntry.tsx

```diff
@@ -1,3 +1,5 @@
+import React from "react";
```

Comment:
The newer versions of React don't need this defined. Normally you would resolve this in ESLint

Reply:
I deleted this on the pages where it was unnecessary, but the pages where I used React TypeScript elements, I let them stay.